docker/test/integration/minifi/processors/DeleteS3Object.py

```diff
@@ -1,13 +1,16 @@
 from ..core.Processor import Processor
 
+
 class DeleteS3Object(Processor):
-    def __init__(self,
-        proxy_host = '',
-        proxy_port = '',
-        proxy_username = '',
-        proxy_password = ''):
-            super(DeleteS3Object, self).__init__('DeleteS3Object',
-            properties = {
+    def __init__(
+            self,
```

Comment:
In most cases, continuation indentation was changed to one level of indentation, but here it's changed to two. Why?

Reply:
Not sure about python guidelines in this regard, but my recommendation would be to stick to two levels of indentation for continuations.

Reply:
why not one level: to clearly separate it from normal code one layer deeper
why not aligned: to support find/replace and rename refactoring without messing up the layout

Reply:
In that case we would have 2 problems with flake8:
./docker/test/integration/minifi/processors/DeleteS3Object.py:10:5: E125 continuation line with same indent as next logical line
./docker/test/integration/minifi/processors/DeleteS3Object.py:11:13: E117 over-indented

Reply:
The [pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#indentation) is the same in this case:
```
# Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)
```

Reply:
Let's go with pep8 in this case